cfme/services/catalogs/catalog.py

```diff
@@ -86,24 +81,100 @@ def create(self):
         flash.assert_success_message(flash_str.format(self.name))
 
     def update(self, updates):
-        sel.force_navigate('catalog_edit', context={'catalog': self})
+        navigate_to(self, 'Edit')
         web_ui.fill(form, {'name_text': updates.get('name', None),
                            'description_text': updates.get('description', None),
                            'button_multiselect': updates.get('items', None)},
                     action=form.save_button)
         flash.assert_success_message('Catalog "{}" was saved'.format(self.name))
 
     def delete(self):
-        sel.force_navigate('catalog', context={'catalog': self})
-        cfg_btn("Remove Item from the VMDB", invokes_alert=True)
+        navigate_to(self, 'Details')
+        cfg_btn(version.pick(cfg_details_remove), invokes_alert=True)
```

Comment:
this is the only place where you have used this text. I'd be better to move it here.